src/components/DropDown/DropDown.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './DropDown.css';
+const DropDown = () => {
```

Comment:
I think there should be a `<options>` in the dropdown or something to get the value of the clicked item from! 
or you can add each `li` as a button with onClick